includes/definitions/discovery/vrp.yaml

```diff
@@ -119,4 +119,35 @@ modules:
                     index: '{{ $index }}'
                     skip_values: -1
                     divisor: 1000000
-
+        load:
+            data:
```

Comment:
You probably want "fan" and not "load" here :)

Reply:
there is value in percentage instead of rpm, so we cant use fan category.

Reply:
![image](https://user-images.githubusercontent.com/36922215/52960131-60960200-3398-11e9-8677-312927352a75.png)

Reply:
If you have the full speed in an OID, you can do the math in yaml and convert to the proper unit (RPM). If you don't, then you should probably use count and a group to explain what it is.

Reply:
I found some of the maximum speed of the fan, but there is more than one module of fan, with different speeds, and i cant find an oid with that information for now. For VRP this will be the very first load sensor, and look at the picture in description i believe that its clear about what is going on. I can add group, this is not a problem.

Reply:
If it is not exactly what the load class is for, then we must use "count" which is the only generic class we have. Then groups and name of the sensor will explain the user what it is.

Reply:
Can't use count here.  You could add a new sensor for this, otherwise, drop it.

Reply:
@murrant  i dont know how to add new sensor, so i dropped load of the fan.